src/main/java/christmas/domain/plan/MenuCategory.java

```diff
@@ -0,0 +1,63 @@
+package christmas.domain.plan;
+
+import java.util.Arrays;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+import static christmas.domain.plan.CategorizedMenuConstant.*;
+
+public enum MenuCategory {
+
+    APPETIZER("애피타이저", APPETIZER_MENUS),
+    MAIN("메인", MAIN_MENUS),
+    DESSERT("디저트", DESSERT_MENUS),
+    DRINK("음료", DRINK_MENUS);
+
+    private final String category;
+    private final List<Menu> menus;
+
+    MenuCategory(String category, List<Menu> menus) {
+        this.category = category;
+        this.menus = menus;
+    }
+
+
+    public static MenuCategory findCategoryByMenu(Menu menu) {
+        return Arrays.stream(MenuCategory.values())
+                .filter(category -> category.menus.contains(menu))
+                .findFirst().get();
```

Comment:
분명 알고계실 것 같은 내용이지만, 노파심에 코멘트 드립니다!

현재의 코드는 차한님이 직접 구현하셨고 `findCategoryByMenu`에 존재하지 않는 메뉴를 입력하지 않을테니 문제가 생기진 않겠지만, 그럼에도 코드 안정성을 위해서라면 `Optional`에서 값을 가져 올 때 값이 없는 경우 예외를 명시적으로 던져주는 것이 좋다고 생각해요!
```suggestion
                .findFirst()
                .orElseThrow(IllegalArgumentException::new);
```